utilities/utils/utils.go

```diff
@@ -51,6 +51,9 @@ func IsNoOp(data model.ProcessData) bool {
 }
 
 func AddLabel(config *container.Config, key string, value string) {
+	if config.Labels == nil {
```

Comment:
ah i got a panic in a test, but i think that is bc the test is not end to end. its a unit test for the function. i'll tweak the test to init that map